biz.aQute.bndlib.tests/test/aQute/bnd/build/repo/WorkspaceRepositoryTest.java

```diff
@@ -46,6 +47,77 @@ public void findprovidersMacroTest() throws Exception {
 		});
 	}
 
+	@Test
+	public void findprovidersMacroTestWithStrategy() throws Exception {
+
+		test(ws -> {
+			Project project = ws.getProject("p1");
+			assertThat(project).isNotNull();
+			File[] build = project.build();
+			assertThat(build).isNotNull()
+				.isNotEmpty();
+			assertThat(build[0]).hasName("p1.jar");
+			String services = ws._findproviders(new String[] {
```

Comment:
I don't think we want tests to directly call the macro method. That bypasses actually testing the use of the macro and the parsing of parameters, etc.